java/videostore/src/VideoStoreTest.java

```diff
@@ -1,38 +1,61 @@
-		customer.addRental (new Rental (new Movie ("The Tigger Movie", Movie.CHILDRENS), 3));
-		assertEquals ("Rental Record for Fred\n\tThe Tigger Movie\t1.5\nYou owed 1.5\nYou earned 1 frequent renter points\n", customer.statement ());
-	}
-
-	public void testMultipleRegularStatement () {
-		customer.addRental (new Rental (new Movie ("Plan 9 from Outer Space", Movie.REGULAR), 1));
-		customer.addRental (new Rental (new Movie ("8 1/2", Movie.REGULAR), 2));
-		customer.addRental (new Rental (new Movie ("Eraserhead", Movie.REGULAR), 3));
-
-		assertEquals ("Rental Record for Fred\n\tPlan 9 from Outer Space\t2.0\n\t8 1/2\t2.0\n\tEraserhead\t3.5\nYou owed 7.5\nYou earned 3 frequent renter points\n", customer.statement ());
-	}
-
-	private Customer customer;
-}
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class VideoStoreTest {
+
+    private Customer customer;
+    private Movie newReleaseMovie1;
+    private Movie regularMovie3;
+    private Movie newReleaseMovie2;
+    private Movie regularMovie1;
+    private Movie newChildrensMovie;
+    private Movie regularMove2;
+
+    @Before
+    public void setUp() {
+        customer = new Customer("Fred");
+        newReleaseMovie1 = new Movie("The Cell", Movie.NEW_RELEASE);
+        newReleaseMovie2 = new Movie("The Tigger Movie", Movie.NEW_RELEASE);
+        newChildrensMovie = new Movie("The Tigger Movie", Movie.CHILDRENS);
+        regularMovie1 = new Movie("Plan 9 from Outer Space", Movie.REGULAR);
+        regularMove2 = new Movie("8 1/2", Movie.REGULAR);
+        regularMovie3 = new Movie("Eraserhead", Movie.REGULAR);
+    }
+
+    @Test
+    public void testSingleNewReleaseStatement() {
+        customer.addRental(new Rental(newReleaseMovie1, 3));
+        customer.statement();
+        assertEquals(9.0, customer.getAmountOwed());
+        assertEquals(2, customer.getFrequentRenterPoints());
+    }
+
+    @Test
+    public void testDualNewReleaseStatement() {
+        customer.addRental(new Rental(newReleaseMovie1, 3));
+        customer.addRental(new Rental(newReleaseMovie2, 3));
+        customer.statement();
+        assertEquals(18.0, customer.getAmountOwed());
+        assertEquals(4, customer.getFrequentRenterPoints());
+    }
+
+    @Test
+    public void testSingleChildrensStatement() {
+        customer.addRental(new Rental(newChildrensMovie, 3));
+        customer.statement();
+        assertEquals(1.5, customer.getAmountOwed());
+        assertEquals(1, customer.getFrequentRenterPoints());
+    }
+
+    @Test
+    public void testMultipleRegularStatement() {
+        customer.addRental(new Rental(regularMovie1, 1));
+        customer.addRental(new Rental(regularMove2, 2));
+        customer.addRental(new Rental(regularMovie3, 3));
+        customer.statement();
+        assertEquals(7.5, customer.getAmountOwed());
+        assertEquals(3, customer.getFrequentRenterPoints());
+    }
+}
```

Comment:
Good job on cleaning up the tests. I think you can go a step further by extracting helper functions for creating a new movie and rental. The `new` keyword appears a bit too much in this file.